proto/chainlink/v1beta/query.proto

```diff
@@ -9,17 +9,34 @@ option go_package = "github.com/ChainSafe/chainlink-cosmos/x/chainlink/types";
 
 // Query defines the gRPC querier service.
 service Query {
-    rpc FeedDataByID(QueryFeedDataRequest) returns (QueryFeedDataResponse) {
-        option (google.api.http).get = "/chainlink/v1/feed/data/{feedId}";
-    }
+  rpc GetRoundData(GetRoundDataRequest) returns (GetRoundDataResponse) {
```

Comment:
jeez.. i hate this naming... i think it makes more sense to use this concept:
```
Feed {
  feedId uint, 
  ...
  feedData[roundId],
}
```
so conceptually, it makes more sense to me to call this something like "GetFeedDataByRound", similar to what we had before... i dont know if this makes more sense or not..!

Reply:
I think you are not looking at the latest code man, my latest commit has totally different name